diff --git a/node_modules/native-base/src/core/NativeBaseProvider.tsx b/node_modules/native-base/src/core/NativeBaseProvider.tsx
index 43b4bd1..7f1ae0b 100644
--- a/node_modules/native-base/src/core/NativeBaseProvider.tsx
+++ b/node_modules/native-base/src/core/NativeBaseProvider.tsx
@@ -94,7 +94,8 @@ const NativeBaseProvider = (props: NativeBaseProviderProps) => {
             <OverlayProvider isSSR>
               <ToastProvider>
                 <InitializeToastRef />
-                <SSRProvider>{children}</SSRProvider>
+                {/* <SSRProvider>{children}</SSRProvider> */}
+                {React.version >= '18' ? children : <SSRProvider>{children}</SSRProvider>}
               </ToastProvider>
             </OverlayProvider>
           </HybridProvider>
